### PR TITLE
update pyramid-cookiecutter-* for Pyramid 1.10

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -490,9 +490,9 @@ Python-Django
 Python-Pyramid
 ^^^^^^^^^^^^^^
 
-* `pyramid-cookiecutter-alchemy`_: A Cookiecutter (project template) for creating a Pyramid project using SQLite for persistent storage, SQLAlchemy for an ORM, URL dispatch for routing, and Jinja2 for templating.
-* `pyramid-cookiecutter-starter`_: A Cookiecutter (project template) for creating a Pyramid starter project using URL dispatch for routing and either Jinja2, Chameleon, or Mako for templating.
-* `pyramid-cookiecutter-zodb`_: A Cookiecutter (project template) for creating a Pyramid project using ZODB for persistent storage, traversal for routing, and Chameleon for templating.
+* `pyramid-cookiecutter-alchemy`_: [DEPRECATED - Please use `pyramid-cookiecutter-starter`_ instead] A Cookiecutter (project template) for creating a Pyramid project using SQLite for persistent storage, SQLAlchemy for an ORM, URL dispatch for routing, and Jinja2 for templating.
```
Comment (insspb) [inline README.rst]: Please drop deprecated information from cookiecutter readme. I checked your repos and you have this note there. Why do we need to keep this outdated links in this doc?

File: README.rst [hunk 2]
```diff
@@ -490,9 +490,9 @@ Python-Django
...
+* `pyramid-cookiecutter-starter`_: A Cookiecutter (project template) for creating a Pyramid starter project with choices for template language (Jinja2, Chameleon, or Mako), persistent backend (none, SQLAlchemy with SQLite, or ZODB), and mapping of URLs to routes (URL dispatch or traversal).  As of Pyramid 1.10, this cookiecutter adds features from pyramid-cookiecutter-alchemy and pyramid-cookiecutter-zodb.
+* `pyramid-cookiecutter-zodb`_: [DEPRECATED - Please use `pyramid-cookiecutter-starter`_ instead] A Cookiecutter (project template) for creating a Pyramid project using ZODB for persistent storage, traversal for routing, and Chameleon for templating.
```
Comment (insspb) [inline README.rst]: Please drop deprecated information from cookiecutter readme. I checked your repos and you have this note there. Why do we need to keep this outdated links in this doc?

File: README.rst (hunk 1 above)
Comment (stevepiercy) [reply README.rst]: Done!

File: README.rst (hunk 2 above)
Comment (stevepiercy) [reply README.rst]: Done!

**Comment by ssbarnea** [comment]
We are no longer maintaining a list of templates inside README file and instead we use github topics feature. Please read https://github.com/cookiecutter/cookiecutter#a-pantry-full-of-cookiecutters and add the needed topics to your repository and people will be able to find your template. The most important one is ``cookiecutter-template``.

We are also going to remove most of already listed templates from there, in order not avoid being biased.

**Comment by stevepiercy** [comment]
Added topics for pyramid-* and substanced cookiecutters.

**Pull request closed without merging**